docs/security/host-certs.md

```diff
@@ -205,6 +205,56 @@ For instructions on how to request a host or service certificate using the Web
 interface please see the [user guide maintained by
 the OIM development team](https://confluence.grid.iu.edu/pages/viewpage.action?pageId=3244064).
 
+Requesting Host Certificate for HTCondor-CE Using Let's Encrypt
+---------------------------------------------------------------
+
+As an alternative to the above options, the Let's Encrypt software can be used to obtain host certificates.
+
+We use the following guide as a reference:
+
+   - https://github.com/cilogon/letsencrypt-certificates
+
+In particular, under Getting your host certificate, we follow the two "Setting up" sections.
+
+(See also: https://letsencrypt.org/getting-started/)
+
+The `letsencrypt` software (AKA `certbot`) can be obtained from the EPEL 7 yum repo:
+
+        :::console
+        root@host # yum install certbot
+
+If you already have an HTCondor-CE set up and running, stop the CE View service (as it listens on
+port 80, which the letsencrypt setup needs to bind on temporarily).
```

Comment:
Let's generalize this and say any service running on port 80 and then use the CE View as an example (with a link to the CE View section in the docs)

Reply:
Good point, let's fix up the header or anywhere else that's CE-specific in that case. We can update the JIRA ticket, too